### DISCUSSION TITLE
[Feature Request] New Select Components

### DISCUSSION THREAD

**Discussion by marcozzxx810** [heroui-inc/heroui#1321] [Feature requests]
### Is your feature request related to a problem? Please describe.

N/A

### Describe the solution you'd like

As a user, I would like to have a component similar to `select` in HTML for DropDown selection.

```HTML
<select name="cars" id="cars">
    <option value="volvo">Volvo</option>
    <option value="saab">Saab</option>
    <option value="opel">Opel</option>
    <option value="audi">Audi</option>
 </select>

```

### Describe alternatives you've considered

Although there is a DropDown component, it is more like a dialog box. As a user, we want to have something similar to <select>, so that user can select the option directly. 

### Screenshots or Videos

Similar to https://www.w3schools.com/tags/tryit.asp?filename=tryhtml_select

**Comment by jrgarciadev** [comment]
In progress: https://github.com/nextui-org/nextui/pull/1313

**Reply by marcozzxx810** [reply]
Awesome, looking forward to that!

**Reply by katieatwell** [reply]
I keep checking in for it to be available, so excited it is on the way.
